### PR TITLE
filePaths file for idr0032 datasets

### PR DESCRIPTION
**Pull request by eleanorwilliams**
path to .tif files for idr0032.

We may have to delete a few (<10) as there are some that are not listed in the paper supplementary materials, but the authors may want to include them.  Easier to show the authors which ones are extra if they are in IDR

--exclude

### CODE REVIEW

**Comment by joshmoore** [comment]
Shall we move this out of `/incoming`?

**Comment by eleanorwilliams** [comment]
@joshmoore.  Changed file paths to /uod/idr/filesets.  Left EBI files in /drop/.
Forgot to mention - watch out a few file names have an apostrophe in them e.g. /uod/idr/filesets/idr0032-yang-meristem/20161104-original/Pictures of all GTs/AT5G62220/2'.tif

**Comment by eleanorwilliams** [comment]
@joshmoore This project is now ready for test loading into either demo32 or test32

File: idr0032-yang-meristem/experimentA/idr0032-experimentA-bulk.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+transfer: upload
```
Comment (joshmoore) [inline idr0032-yang-meristem/experimentA/idr0032-experimentA-bulk.yml]: This will need to get reverted for production.

**Comment by eleanorwilliams** [comment]
Removed "transfer: upload" from bulk.yml file. 
Tested annotations in project on test32.  Look ok.

File: idr0032-yang-meristem/experimentA/idr0032-experimentA-bulk.yml (hunk 1 above)
Comment (eleanorwilliams) [inline idr0032-yang-meristem/experimentA/idr0032-experimentA-bulk.yml]: Now removed.

**Comment by joshmoore** [comment]
Merging for import to production (if files are not present under `/uod/idr/metadata` during symlinking chaos ensues)

**Pull request merged**